mix.exs

```diff
@@ -7,7 +7,7 @@ defmodule MIME.Mixfile do
     [
       app: :mime,
       version: @version,
-      elixir: "~> 1.10",
+      elixir: "~> 1.11",
```

Comment:
```suggestion
      elixir: "~> 1.10",
```

I believe we cannot run CI on 1.10 but I would like to avoid changing the minimum version whenever possible.